packages/cinterop/build.gradle.kts

```diff
@@ -88,24 +88,53 @@ android {
     }
 }
 
-val nativeLibraryIncludesMacosUniversal = listOf(
-    "-include-binary", "$cinteropCorePath/capi_macos_universal/src/realm/object-store/c_api/librealm-ffi-static-dbg.a",
-    "-include-binary", "$cinteropCorePath/capi_macos_universal/src/realm/librealm-dbg.a",
-    "-include-binary", "$cinteropCorePath/capi_macos_universal/src/realm/parser/librealm-parser-dbg.a",
-    "-include-binary", "$cinteropCorePath/capi_macos_universal/src/realm/object-store/librealm-object-store-dbg.a"
+val nativeLibraryIncludesMacosUniversalRelease = listOf(
+    "-include-binary",
+    "$absoluteCorePath/capi_macos_universal/src/realm/object-store/c_api/librealm-ffi-static.a",
```

Comment:
Really annoying that the directory `capi_macos_universal` doesn't follow the pattern of all the other build directories by prefixing `build_`